.github/workflows/release-pypi-kernel.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - main
     paths:
-      - sgl-kernel/pyproject.toml
+      - sgl-kernel/version.py
```

Comment:
Avoid releases caused by changes to `pyproject.toml` (we should only release when the version changes).